Makefile

```diff
@@ -33,7 +33,7 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
-GIT_INFO := $(git describe --long --dirty)
+GIT_INFO=$(shell git describe --long --dirty)
```

Comment:
The shift to the `shell` function makes a lot of sense. 

I think if I had done this myself, I wouldn't have swapped `:=` for `=` but that's an instinct thing not a purely rational thing, and I do much less makefile development than I used to.

Reply:
Makes sense. `:=` seems better